template/composer.json

```diff
@@ -15,7 +15,7 @@
         "drupal/cog": "^1.0.0",
         "drupal/core": "^8.0",
         "drupal/features": "^3.0.0",
-        "drupal/memcache": "^2.0.0-alpha2",
+        "drupal/memcache": "2.x-dev",
```

Comment:
Oh I see. I noticed that a patch had changed in this PR and assumed it was for memcache. Looks like it's actually just a whitespace change.